diff -Naur drivers/media/dvb/dvb-core.orig/dvbdev.c drivers/media/dvb/dvb-core/dvbdev.c
--- drivers/media/dvb/dvb-core.orig/dvbdev.c	2011-10-17 17:26:40.840432000 +0900
+++ drivers/media/dvb/dvb-core/dvbdev.c	2011-10-17 17:28:59.360445134 +0900
@@ -417,10 +417,10 @@
 	}
 
 	/* call driver */
-	mutex_lock(&dvbdev_mutex);
+	//mutex_lock(&dvbdev_mutex);
 	if ((err = func(file, cmd, parg)) == -ENOIOCTLCMD)
 		err = -EINVAL;
-	mutex_unlock(&dvbdev_mutex);
+	//mutex_unlock(&dvbdev_mutex);
 
 	if (err < 0)
 		goto out;
